google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -221,18 +217,19 @@ public void run() {
     }
 
     private void stop(@NotNull UndeploymentTaskCallback callback) {
-      String moduleToStop;
+      DeployOutput deployOutput;
       try {
-        moduleToStop = parseDeployOutputToModule(deploymentOutput);
```

Comment:
Yeah that's reasonable.  But if our parsing of the deployment result is
broken, I would prefer that we broke immediately, rather than lazily during
stop.  For one we'd be much more likely to notice the breakage.


On Mon, May 2, 2016 at 10:51 AM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/660#discussion_r61748614>
> :
>
> >        try {
> > -        moduleToStop = parseDeployOutputToModule(deploymentOutput);
>
> Not that performance is really a concern here but just to explain the
> original reason I did this: having it in the stop() method lazily does the
> json parsing (doesn't happen until the user actually tries to undeploy
> which may never happen in the first place).
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/660/files/ddd3d9a477599546aacb5636136c10cad7eebe79#r61748614>
>